packages/react/src/tooltip/OverflowTooltip.js

```diff
@@ -61,7 +62,7 @@ const OverflowTooltip = forwardRef((
   useEventListener(eventTargetFn, 'mouseleave', onMouseLeave);
 
   const tooltipProps = {
-    disabled: !isOverflow,
+    disabled: disabled || !isOverflow,
```

Comment:
Here is the summary:
* If `disabled` is `true`, the `disabled` property in `tooltipProps` will be set to `true`, regardless of `isOverflow`.
* If `disabled` is `false` or `undefined`, the `disabled` property in `tooltipProps` will depend on `isOverflow`:
  * When there is no overflow, the tooltip will be disabled (the `disabled` prop will be `true`).
  * When there is overflow, the tooltip will remain enabled (the `disabled` prop will be `false`).

Reply:
Yes, it is. Thank you for your summary.

Reply:
I modified the code to conditionally set the `disabled` prop based on the overflow state. As for the `nextToCursor` prop, there's no need to check the overflow state. Did you notice any issues with this update?

```js
const tooltipProps = {
  // The `disabled` prop is set to `true` if there is no overflow
  disabled: isOverflow ? disabledProp : true,
  nextToCursor: nextToCursorProp,
  offset: offsetProp,
  placement: placementProp,
};
```